### PR TITLE
fix build warnings task - #337

### PR DESCRIPTION
**Pull request by SlakvichAleksandr**

### CODE REVIEW

**Review by Erbidium — APPROVED** [review]

**Review by tatianahutii — CHANGES_REQUESTED** [review]
Still see warning
image

File: frontend/src/app/modules/settings/user/user-profile-page/user-profile-page.component.ts [hunk 1]
```diff
@@ -186,4 +186,8 @@ export class UserProfilePageComponent extends BaseComponent implements OnInit {
     public userNameChanged(value: string) {
         this.userForm.patchValue({ userName: removeExcessiveSpaces(value) });
     }
+
+    public defineZone() {
+        return !this.user?.timeZone || (this.user.timeZone.nameValue === '' && this.user.timeZone.timeValue === '');
```
Comment (tatianahutii) [inline frontend/src/app/modules/settings/user/user-profile-page/user-profile-page.component.ts]:
```suggestion
        return !this.user?.timeZone || (!this.user.timeZone.nameValue&& !this.user.timeZone.timeValue);
```

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/package.json [hunk 2]
```diff
@@ -66,6 +66,7 @@
         "karma-jasmine-html-reporter": "~1.7.0",
         "prettier": "^2.7.1",
         "prettier-eslint": "^15.0.1",
-        "typescript": "^4.6.4"
+        "typescript": "^4.6.4",
+        "webpack": "^5.74.0"
```
Comment (tatianahutii) [inline frontend/package.json:70]: Why did you add this?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/angular.json [hunk 3]
```diff
@@ -59,8 +60,8 @@
                                 },
                                 {
                                     "type": "anyComponentStyle",
-                                    "maximumWarning": "6kb",
-                                    "maximumError": "10kb"
+                                    "maximumWarning": "10kb",
+                                    "maximumError": "15kb"
```
Comment (tatianahutii) [inline frontend/angular.json]: do not change angular.json

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 2 above)
Comment (SlakvichAleksandr) [reply frontend/package.json:70]: Added for fix - warning "@angular/fire > file-loader@6.2.0" has unmet peer dependency "webpack@^4.0.0 || ^5.0.0". Without this package we have warning

File: frontend/angular.json (hunk 3 above)
Comment (SlakvichAleksandr) [reply frontend/angular.json]: We can't decrease size of schedule-definition.component.sass. Any extra imports or unused styles. For this reason budgets was changed

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]
